index.html

```diff
@@ -309,11 +309,30 @@
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys. The [=controller document=] contains [=verification
-relationships=] that explicitly permit the use of certain [=verification
-methods=] for specific purposes.
+A [=controller document=] contains cryptographic material and service
+endpoints for the purposes of verifying proofs from, and interacting
+with, the [=controller=] of an identifier.
+      </p>
+      <p>
```

Comment:
This paragraph repeats the (current) Abstract, word-for-word, and should be deleted!
```suggestion
```

Reply:
But if retained, the rephrasing of the Abstract should also be applied here.
```suggestion
A [=controller document=] contains cryptographic material and identifies
service endpoints that can be used to verify proofs from, and interact
with, the [=controller=] of an identifier.
      </p>
      <p>
```